metrics/cli/lib/util/dependencies/dependency.dart

```diff
@@ -0,0 +1,36 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:cli/services/common/info_service.dart';
+import 'package:equatable/equatable.dart';
+
+/// A class that represents the information on a specific [InfoService].
+class Dependency extends Equatable {
+  /// A [String] representing the recommended version of this dependency.
+  final String recommendedVersion;
+
+  /// A [String] representing the installation URL of this dependency.
+  final String installUrl;
+
+  @override
+  List<Object> get props => [recommendedVersion, installUrl];
+
+  /// Creates a new instance of [Dependency] with the given [recommendedVersion]
+  /// and [installUrl].
+  const Dependency._({
```

Comment:
🔨 
Not sure this constructor is to be private.
```suggestion
  const Dependency({
```